src/clients/FlexbaseClient.Patron.ts

```diff
@@ -1,6 +1,21 @@
 import { Patron } from '../models/Patron/Patron';
 import { FlexbaseClientBase } from './FlexbaseClient.Base';
 
+interface IPatronData {
```

Comment:
moving away from using `I` on interfaces so just have `PatronData`

Reply:
I got it